scripts/start.py

```diff
@@ -75,12 +75,15 @@
 
 
 def cleanup():
-    """Function for waiting for the servers to go down."""
+    """Wait for the servers to go down and set constants back to default
+    values.
+    """
     common.print_each_string_after_two_new_lines([
         'INFORMATION',
         'Cleaning up the servers.'])
     while common.is_port_open(PORT_NUMBER_FOR_GAE_SERVER):
         time.sleep(1)
+    build.set_constants_to_default()
```

Comment:
This doesn't explain which constants, can you please add a code comment*

Reply:
Hmm, I do not think that we can list them here since they might change and this comment can actually only become confusing if some developer needs to know which are changed they can look into the function definition. but I think I can add a general sounding comment explaining that this is done in order to keep the files unmodified after starting the server.